### PR TITLE
Station-shuttlesystem interactions are now properly implemented.

### PR DESCRIPTION
**Pull request by monster860**

### CODE REVIEW

**Comment by monster860** [comment]
God damn it, git.....

**Comment by monster860** [comment]
Damn, it wasn't git. It was mapmerge or DM being an ass and not deleting unused keys.

File: code/modules/shuttle/shuttle.dm [hunk 1]
```diff
@@ -466,11 +475,23 @@
...
+				if(nA)
+					nA.contents += T1
+				new cutout_newturf(T1)
 		if(!T0)
 			continue
-		var/turf/T1 = L1[i]
 		if(!T1)
 			continue
+		var/transfer_area = 1
+		if(T1.loc.type == cutout_extarea)
```
Comment (CrAzYPiLoT-SS13) [inline code/modules/shuttle/shuttle.dm]: Why not `istype`?

File: code/modules/shuttle/shuttle.dm [hunk 2]
```diff
@@ -466,11 +475,23 @@
 
 	for(var/i in 1 to L0.len)
 		var/turf/T0 = L0[i]
+		var/turf/T1 = L1[i]
+		if(!T0 && T1)
+			var/turf/T0_all = L0_all[i]
+			if(T0_all.loc.type == cutout_extarea)
```
Comment (CrAzYPiLoT-SS13) [inline code/modules/shuttle/shuttle.dm]: Other comment.

File: code/modules/shuttle/shuttle.dm (hunk 1 above)
Comment (monster860) [inline code/modules/shuttle/shuttle.dm]: The subtype is not something I want to include.

**Pull request merged**